### DISCUSSION TITLE
Connecting to my database from outside?

### DISCUSSION THREAD

**Discussion by oOoBlackHoleSunoOo** [moneymanagerex/moneymanagerex#6163] [Q&A]
Hello, 
First of all like MME for easy handling of my accounts with a easy UI and somewhat speed. However Report customizing is a bit annoying.

PROBLEM:
The General Report SQL Window is crashing very often on requests and its annoying .Also the General  Report UI is very small and i have no clue of LUA for further editing. I am more the python, JavaScript guy. So i thought for special occasions it would be nice to write my own little report scripts in python or JavaScript  to get things faster done.

QUESTION:
Is it possible to connect to my database from the outside?  Is Money Manager Ex initiating a database server while running? What is the IP/Port?

**Comment by tactilis** [comment]
> Also the General Report UI is very small 

You can resize the window to be as large as you wish.


>  Is Money Manager Ex initiating a database server while running? 

No.

However, the .mmb file is simply an SQLite database. You can open it (when MMEX is not using it) with, for example:
- DB Browser https://sqlitebrowser.org/
- SQLite Studio https://sqlitestudio.pl/

Or any app that can handle SQLite.

<br>

> The General Report SQL Window is crashing very often on requests and its annoying

Please open a new issue giving specific details of the crashes so they can be investigated. This is the _only_ way that the problem can be diagnosed and the annoyance for you removed.

**Reply by altinturk** [reply]
Hi @tactilis - totally true, just tried an unencrypted database and it is easy to read it as sqlite-
do you have an idea of how to connect to an encrypted one?

**Comment by vomikan** [comment]
https://stackoverflow.com/questions/64435111/how-can-i-add-an-sqlite-database-to-apache-superset

**Comment by vomikan** [comment]
You can overtake this one from sqlite to duckDB, it's a good alternative.
